gulpfile.js

```diff
@@ -38,16 +38,18 @@ function getBundleConfig() {
 
     if (RELEASE) {
         config.plugins = config.plugins.concat(
-            new webpack.optimize.UglifyJsPlugin(),
-            new webpack.optimize.OccurenceOrderPlugin()
+            new webpack.optimize.UglifyJsPlugin({
+                sourceMap: true
+            }),
+            new webpack.optimize.OccurrenceOrderPlugin()
         );
     } else {
         config.output.pathinfo = true;
     }
 
-    if (!RELEASE || DEBUG) {
-        config.devtool = '#inline-source-map';
-    }
+    config.devtool = (RELEASE)
+                        ? 'source-map'
```

Comment:
It's not a big deal - we can definitely leave it the way it is ☺️ I'd just do a single tab indent though:

```js
config.devtool = (RELEASE)
    ? 'source-map'
    : 'cheap-module-eval-source-map';
```